geode-docs/reference/topics/gemfire_properties.html.md.erb

```diff
@@ -518,6 +518,17 @@ Any security-related (properties that begin with <code class="ph codeph">securit
 <td><em>not set</em></td>
 </tr>
 <tr>
+<td>security-auth-token-enabled-components</td>
+<td>A comma delimited list of component names which works in conjunction with the <code class="ph codeph">security-manager</code> property. if security manager is enabled, this property will determine what components will use token based authentication instead of basic (username/password) authentication.
+<p>Valid values are: "all", "management", "pulse"</p>
```

Comment:
do literal quotes work here or do you need to use &quot;

Reply:
they show up correctly in the generated docs